toolkit/scripts/toolchain/container/toolchain_build_in_chroot.sh

```diff
@@ -904,11 +904,11 @@ popd
 rm -rf patch-2.7.6
 touch /logs/status_patch_complete
 
-echo Man-DB-2.8.4
-tar xf man-db-2.8.4.tar.xz
-pushd man-db-2.8.4
+echo Man-DB-2.10.1
+tar xf man-db-2.10.1.tar.xz
+pushd man-db-2.10.1
 ./configure --prefix=/usr                        \
-            --docdir=/usr/share/doc/man-db-2.8.4 \
+            --docdir=/usr/share/doc/man-db-2.10.1 \
```

Comment:
Please also update the `--with-systemdsystemunitdir=` flag here as well, so we build with the same options in both places.

Reply:
Thanks for catching that. Added it now.